### PR TITLE
Jdbc data access and Transactions

### PR DESCRIPTION
**Pull request by aayushi93**
JDBC Data Access Ticket URL - https://www.notion.so/Data-Access-68105df96ed7499ab752f51295fc35ef
Transactions Ticket URL - https://www.notion.so/Transactions-6d9fd2a9ff3d46418b915a448dd52fdc

### CODE REVIEW

File: core_java/.idea/$PRODUCT_WORKSPACE_FILE$ [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
Comment (Jarvis-deprecated-account) [inline core_java/.idea/$PRODUCT_WORKSPACE_FILE$]: do not commit `.idea` files. Please remove

File: core_java/jdbc/pom.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<project>
```
Comment (Jarvis-deprecated-account) [inline core_java/jdbc/pom.xml:1]: do not commit .class file. please remove

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+            statement.setString(3, dto.getEmail());
+            statement.setString(4, dto.getPhone());
+            statement.setString(5, dto.getAddress());
+            statement.setString(6, dto.getCity());
+            statement.setString(7, dto.getState());
+            statement.setString(8, dto.getZipCode());
+            statement.execute();
+            int id = this.getLastVal(CUSTOMER_SEQUENCE);
+            return this.findById(id);
+        }catch(SQLException sqle) {
```
Comment (Jarvis-deprecated-account) [inline core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java]: use `SQLException ex` or `SQLException e`
this is convention

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+            sqle.printStackTrace();
```
Comment (Jarvis-deprecated-account) [inline core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java]: do not use printStacktrack

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/DatabaseConnectionManager.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+public class DatabaseConnectionManager {
+    private final String url;
+    private final Properties properties;
+
+    public DatabaseConnectionManager(String host, String databaseName,
+                                     String username, String password) {
+        this.url = "jdbc:postgresql://"+host+"/"+databaseName;
+        this.properties = new Properties();
+        this.properties.setProperty("user", username);
+        this.properties.setProperty("password", password);
```
Comment (Jarvis-deprecated-account) [inline core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/DatabaseConnectionManager.java:17]: do not hardcode password, use env var

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JdbcExecutor.java [hunk 6]
```diff
@@ -0,0 +1,80 @@
+package ca.jrvs.apps.jdbc;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class JdbcExecutor{
+    public static void main(String[] args) {
+        DatabaseConnectionManager dcm = new DatabaseConnectionManager("localhost", "hplussport",
```
Comment (Jarvis-deprecated-account) [inline core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JdbcExecutor.java:9]: do not hard code. use env var please

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JdbcExecutor.java [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+            customerDAO.create(customer);
+
+             */
+            //Finding customer by id
+            /*
+            Customer customer = customerDAO.findById(1001);
+            System.out.println(customer.getFirstName() + " " + customer.getLastName());
+
+             */
```
Comment (Jarvis-deprecated-account) [inline core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JdbcExecutor.java]: remove commented code please

**Review by ghost — CHANGES_REQUESTED** [review]
PR is rejected. please fix all the comments and open a PR again.

File: core_java/.idea/$PRODUCT_WORKSPACE_FILE$ (hunk 1 above)
Comment (aayushi93) [reply core_java/.idea/$PRODUCT_WORKSPACE_FILE$]: Fixed!

File: core_java/jdbc/pom.xml (hunk 2 above)
Comment (aayushi93) [reply core_java/jdbc/pom.xml:1]: Fixed!

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java (hunk 3 above)
Comment (aayushi93) [reply core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java]: Fixed

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java (hunk 4 above)
Comment (aayushi93) [reply core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java]: Fixed

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JdbcExecutor.java (hunk 7 above)
Comment (aayushi93) [reply core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JdbcExecutor.java]: Fixed

**Pull request closed without merging**